src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -53,12 +57,20 @@ public override async Task OnConnectedAsync(HubConnectionContext connection)
                     hubActivator.Release(hub);
                 }
             }
+            finally
+            {
+                await scope.DisposeAsync();
```

Comment:
Will this be converted back to `async using`?

Reply:
Not unless we make a wrapper struct. Current plan is to leave it as is.